bluetooth/src/androidLibMain/kotlin/UUID.kt

```diff
@@ -0,0 +1,29 @@
+package com.splendo.kaluga.bluetooth
+
+import android.os.ParcelUuid
+
+actual data class UUID(val uuid: ParcelUuid) : BaseUUID {
+
+    override val uuidString: String
+        get() = uuid.toString()
+
+    override val isValid: Boolean
+        get() = true
```

Comment:
Why it's guaranteed to be always valid?

Reply:
Its initialized using a ParcelUUID, which is always a valid UUID. The is valid is only there so that there is a validation check if a UUID can be created from an unsafe source

Reply:
I'd consider switching to typealias and extension methods though.